pkg/approver/manager/predicate/predicate_test.go

```diff
@@ -36,8 +36,13 @@ import (
 )
 
 func Test_RBACBound(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.TODO())
```

Comment:
So we're using the functionality that Tim added in https://github.com/jetstack/cert-manager/pull/4335 to ensure that the webhook server gets stopped when these tests finish 👍🏼